### DISCUSSION TITLE
Need `rtl` class name for next tailwind update

### DISCUSSION THREAD

**Discussion by 1mehdifaraji** [tailwindlabs/tailwindcss#13637] [Ideas]
In almost every `tailwindcss` project I develop, I have to add a custom class name called `.rtl` in global styles myself because `tailwindcss` simply doesn't have a class name for `direction: rtl;` style.
```
.rtl {
  direction: rtl !important;
}
```
By the way the class name `flex-row-reverse` may reverse the row items, but it can't reverse text content for example arabic or persian text or it can't reverse the direction for all of the elements inside the items.

I think `tailwindcss` next update should include a simple class name called `rtl` or `dir-rtl`, it's very useful and important.

**Comment by adamwathan** [comment]
Hey! We haven't added this because it's generally discouraged in favor of the `dir` attribute:

https://developer.mozilla.org/en-US/docs/Web/CSS/direction
